package.json

```diff
@@ -25,7 +25,9 @@
     "terser-webpack-plugin": "2.2.1",
     "updates": "9.3.0",
     "webpack": "4.41.2",
-    "webpack-cli": "3.3.10"
+    "webpack-cli": "3.3.10",
+    "fomantic-ui": "2.8.3",
```

Comment:
Maybe move it to `dependencies` given that it is code actually used on the frontend.